libs/s25main/figures/nofMiner.h

```diff
@@ -5,11 +5,12 @@
 #pragma once
 
 #include "nofWorkman.h"
+#include "addons/AddonMiningOverhaul.h"
+#include <gameData/GameConsts.h>
```

Comment:
Every include slows down compilation, and includes in headers even more --> Transitive includes: Everyone including this header now also includes those headers so any change in e.g. GameConsts.h now requires recompiling this class and every file including this header although this header does not depend on `GameConsts.h`, does it?

TLDR: Be cautious about adding includes, especially to headers. See if it is enough to add them to the cpp files (it is here for the GameConsts.h, but not for the addons file)

Reply:
Allright, never put that into consideration. Don't know why, but I had in mind to always add the includes to the ```*.h``` file. Which makes no sense at all I guess, or rather has no use.